artifact-version-diff/src/main/resources/3.20_allowed_artifacts.yaml

```
@@ -0,0 +1,27 @@
+allowed-artifacts-version-comparison:
+  # https://issues.redhat.com/browse/QUARKUS-4089
+  - artifact:
+      org.slf4j:slf4j-api
+    allowed-rhbq-versions:
+      - 2.0.7.redhat-00003
+  # Kafka downgrade https://issues.redhat.com/browse/QUARKUS-5849
+  - artifact:
+      org.apache.kafka:kafka-streams-test-utils
+    allowed-rhbq-versions:
+      - 3.7.1
+  - artifact:
+      org.apache.kafka:kafka*
+    allowed-rhbq-versions:
+      - 3.7.1.redhat-00001
+allowed-missing-artifacts-bom-comparison:
+  # https://issues.redhat.com/browse/QUARKUS-6176 https://issues.redhat.com/browse/QUARKUS-6177
+  - com.aayushatharva.brotli4j:native-linux-riscv64
+  - com.aayushatharva.brotli4j:native-windows-aarch64
+  - org.bouncycastle:bctls-jdk18on
```

```suggestion
  # https://issues.redhat.com/browse/QUARKUS-6176
  - com.aayushatharva.brotli4j:native-linux-riscv64
  - com.aayushatharva.brotli4j:native-windows-aarch64
  # https://issues.redhat.com/browse/QUARKUS-6177
  - org.bouncycastle:bctls-jdk18on
```

Please add it inline for easier orientation.